piperider_cli/cli.py

```diff
@@ -220,3 +221,40 @@ def compare_reports(**kwargs):
     CompareReport.exec(a=a, b=b, last=last, datasource=datasource,
                        report_dir=kwargs.get('report_dir'), output=kwargs.get('output'),
                        debug=kwargs.get('debug', False))
+
+
+@cli.group('cloud', short_help='Manage PipeRider Cloud', cls=BetaGroup)
+def cloud(**kwargs):
+    # Manage PipeRider Cloud.
+    pass
+
+
+@cloud.command(short_help='Upload a report to the PipeRider Cloud.', beta=True, cls=TrackCommand)
+@click.option('--run', type=click.Path(exists=True), help='Specify the raw result file.')
+@click.option('--report-dir', default=None, type=click.STRING, help='Use a different report directory.')
+@click.option('--datasource', default=None, type=click.STRING, metavar='DATASOURCE_NAME',
+              help='Specify the datasource.')
+@add_options(debug_option)
+def upload_report(**kwargs):
+    """
+    Upload a single run report to PipeRider Cloud
```

Comment:
I suggest we should remove `single`

Reply:
I will change to use multiple select to let users upload multiple files.

Reply:
Sounds great!

Reply:
Now, users can select multiple reports to upload. Also refactor the report selection function used by compare-reports.